packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -60,8 +60,11 @@ class ScaleStartDetails {
   ///    coordinates.
   final Offset localFocalPoint;
 
+  /// The number of the pointers in contact with the screen.
+ final int pointerCount;
```

Comment:
"number of pointers" rather than "number of the pointers". The former means the count, the latter would mean something like their IDs or something.

Reply:
Done.